pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 dynamic = ["dependencies"]
 
 [project.optional-dependencies]
-test = ["pytest", "cmd2_ext_test"]
+test = ["pytest", "cmd2_ext_test", "gnureadline; platform_system != 'Windows'"]
```

Comment:
let's add this to the main requirements as well as the `pyreadline` so avoid annoying warning while using the CLI

Reply:
you get pip error when you are trying to install pyreadline on unix os and gnureadline on windows os...